include/tins/ip.h

```diff
@@ -281,6 +281,11 @@ class TINS_API IP : public PDU {
 
     /* Getters */
 
+    uint32_t advertised_size() const {
+        uint32_t sz = static_cast<uint32_t>(tot_len());
```

Comment:
Let's just make this a single return, no need for an intermediate variable.

Reply:
Got it.

Reply:
Made the changes, just waiting for your continuous integration.